Lib/ufoLib/glifLib.py

```diff
@@ -454,9 +458,9 @@ def glyphNameToFileName(glyphName, glyphSet):
 	Wrapper around the userNameToFileName function in filenames.py
 	"""
 	if glyphSet:
-		existing = [name.lower() for name in list(glyphSet.contents.values())]
+		existing = set(name.lower() for name in glyphSet.contents.values())
```

Comment:
to be picky, since it's not meant to be modified a frozenset would be more appropriate, but I don't care.